command/host_create.go

```diff
@@ -29,8 +30,16 @@ func (c *createHost) ParseParams(params map[string]interface{}) error {
 
 	// background default is true
 	if _, ok := params["background"]; !ok {
-		params["background"] = true
+		c.CreateHost.Background = true
 	}
+
+	// if a filename is defined, get the params from the file
+	if params["file"] != nil {
```

Comment:
I would do `_, ok := params["file"]; !ok`, it's better style and also sometimes interfaces are weird and it might just be safer.

Reply:
I don't think this works in this case since it's a string and not a boolean. Is there another way to do it so that it's safer?

Reply:
Actually, I don't know if that's the reason why it doesn't work. But changing it to that makes the tests fail with `file '<nil>' does not exist`. I'll look into it more.

Reply:
So with maps you can either do `val := map[key]` or `val, valExists := map[key]` so you can know if the value exists in the map regardless of what the value is (which is why we have _, ok). Theoretically checking the interface would be the same but we've been burned by interfaces before.

Reply:
Thank you! That's definitely good to keep in mind. I'll change it.